### PR TITLE
Add CRUD tests for Environment entity for API part

### LINKED ISSUE #3076: Create CRUD tests for Environment entity in API

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/api/test_environment.py
..........
----------------------------------------------------------------------
Ran 10 tests in 79.076s

OK
```

Closes #3076

### CODE REVIEW

File: tests/foreman/api/test_environment.py [hunk 1]
```diff
@@ -13,38 +13,81 @@
...
-        """@Test: Create an environment and provide an invalid name.
+    def test_positive_create_with_org(self):
+        """@Test: Create an environment and assign it to new organization.
 
-        @Assert: The server returns an error.
+        @Feature: Environment
+
+        @Assert: The environment created successfully and has expected
+        attributes.
+
```
Comment (elyezer) [inline tests/foreman/api/test_environment.py]: You can drop this blank line

File: tests/foreman/api/test_environment.py [hunk 2]
```diff
@@ -13,38 +13,81 @@
...
+    @tier1
+    @run_only_on('sat')
+    def test_positive_create_with_location(self):
+        """@Test: Create an environment and assign it to new location.
+
+        @Feature: Environment
+
+        @Assert: The environment created successfully and has expected
+        attributes.
+
```
Comment (elyezer) [inline tests/foreman/api/test_environment.py]: Same

**Comment by elyezer** [comment]
ACK, pending nitpicks.

**Comment by omaciel** [comment]
ACK. The nitpicks can be handled either now or later. If I don't hear back from @oshtaier in about 15-20m I'll merge it.

**Pull request merged**